app/move/fields/update.js

```diff
@@ -0,0 +1,23 @@
+const { cloneDeep } = require('lodash')
+const createFields = require('./create')
+
+const updateFields = cloneDeep(createFields)
+
+updateFields.police_national_computer = {
+  ...updateFields.police_national_computer,
+  updateProtect: true,
+  updateComponent: {
```

Comment:
Could we make use of the existing `component` property?

Reply:
It's conditional, so these values are there to be copied on to component if required. Personally, I'd rather have both components in the step set up with conditions that determine whether or not to show them. But that's not really the way HMPO rolls

Reply:
[The implementation](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/master/common/templates/form-wizard.njk#L61-L63) of using components isn't anything to do with HMPO. 

We have created that within our service so we can adapt it if we need to.

It's worth being aware that we're not using the full ["Home Office Forms"](https://github.com/UKHomeOfficeForms) implementation which is quite prescriptive for other things like rendering. We are only using the [Form Wizard](https://github.com/UKHomeOffice/passports-form-wizard) to handle the multi step journeys (and now also edit).

Reply:
As discussed, I think it's cleaner to determine which component get output in the controller rather than in the template. Especially as all the other update logic has now moved there too.